packages/react-components/react-storybook-addon/src/preset/manager.ts

```diff
@@ -12,6 +13,13 @@ addons.register(ADDON_ID, () => {
     match: ({ viewMode }) => !!(viewMode && viewMode.match(/^(story|docs)$/)),
     render: ThemePicker,
   });
+  addons.add(DIR_ID, {
+    title: 'Dir Switch',
```

Comment:
it would be good to provide human readable names, Direction rather `Dir`

Reply:
Will be fixed in follow up PR